cherab/core/tests/test_lineshapes.py

```diff
@@ -296,7 +296,7 @@ def test_stark_broadened_line(self):
         line = Line(deuterium, 0, (6, 2))  # D-delta line
         target_species = self.plasma.composition.get(line.element, line.charge)
         wavelength = 656.104
-        integrator = GaussianQuadrature(relative_tolerance=1.e-5)
+        integrator = GaussianQuadrature(relative_tolerance=1.e-8)
```

Comment:
`1.e-8` is used in the `quad` calls too, please consider using a variable instead (e.g. `relative_tolerance`), if the values are supposed to be equal here and below. It could save some trouble in the future.